### DISCUSSION TITLE
Rhino Inside Revit Problem

### DISCUSSION THREAD

**Discussion by SpaceArchitecture** [Tunny-gh/Tunny_in_Open_Sea#334] [General]
[RhinoDotNetCrash.txt](https://github.com/user-attachments/files/17559324/RhinoDotNetCrash.txt)
Hello there, it seems Tunny has a problem with Rhino Inside Revit.

My log report is attached here, what do you think ?

**Comment by hrntsm** [comment]
This file alone does not tell me what is going on.
If you could describe it in more detail, it might give us some idea of what caused it.
